napari/layers/base/base.py

```diff
@@ -1295,6 +1291,36 @@ def _world_to_displayed_data_ray(
         vector_data_ndisplay /= np.linalg.norm(vector_data_ndisplay)
         return vector_data_ndisplay
 
+    def _world_to_data_dims_displayed(
+        self, dims_displayed: List[int], ndim_world: int
+    ) -> List[int]:
```

Comment:
not blocking but should we consider renaming this to `_world_to_displayed_data_dims` be more consistent with other 'world to scene' type methods before merge?

https://github.com/napari/napari/blob/1f68ec2022c35f661fb2e4a704a67925aabb5ba8/napari/layers/base/base.py#L1218-L1239

Reply:
Good question. My interpretation is the naming convention is `<starting coord system>_<ending coord system>_<thing being transformed>`. For example, in the method you linked to above, it takes coordinates from world to the displayed data coordinates. Similarly, `layer.world_to_data_ray` takes a ray from world to data coordinates. In that frame, I named this method `_world_to_data_dims_displayed` since it is taking `dims_displayed` from world to data coordinates. Note that this is not "displayed data" coordinates referenced in `_world_to_displayed_data()`. 

That being said, I don't feel strongly about the name. What do you think? If it wasn't clear that the function is taking `dims_displayed` from world to data coordinates, perhaps I should refine the docstring.

Reply:
discussed offline - let's leave this as is!

Reply:
> In that frame, I named this method `_world_to_data_dims_displayed` since it is taking `dims_displayed` from world to data coordinates. Note that this is not "displayed data" coordinates referenced in `_world_to_displayed_data()`.

Wow, things are getting messy :D

Reply:
Yeah, we can definitely improve the naming convention a bit. Happy to hear suggestions. Since this is consistent with the rest of the codebase, I would suggest we proceed as is and then rename everything once the nomenclature is decided in #3848.